contracts/staking/contracts/src/fees/MixinExchangeFees.sol

```diff
@@ -81,49 +81,46 @@ contract MixinExchangeFees is
             return;
         }
 
-        // Look up the pool for this epoch.
+        // Cache current epoch to reduce sloads.
         uint256 currentEpoch_ = currentEpoch;
-        mapping (bytes32 => IStructs.ActivePool) storage activePoolsThisEpoch =
-            _getActivePoolsFromEpoch(currentEpoch_);
 
-        IStructs.ActivePool memory pool = activePoolsThisEpoch[poolId];
-
-        // If the pool was previously inactive in this epoch, initialize it.
-        if (pool.feesCollected == 0) {
+        // Perform some initialization if this is the pool's first protocol fee in this epoch.
+        uint256 feesCollectedByPool = poolStatsByEpoch[poolId][currentEpoch_].feesCollected;
```

Comment:
Can't we do a `IStructs.PoolStats storage poolStatsThisEpoch = poolStatsByEpoch[poolId][currentEpoch_]` so we aren't doing a double `sload` every time we access it?

Reply:
This is just a single `sload`. Adding more dimensions just changes the storage offset hash. But it could be more readable so I'm down.